etc/packetbeat.yml

```diff
@@ -36,6 +36,26 @@ interfaces:
 
 ############################# Protocols ######################################
 protocols:
+  dns:
+    # Configure the ports where to listen for DNS traffic. You can disable
+    # the DNS protocol by commenting out the list of ports.
+    ports: [53]
+
+    # include_authorities controls whether or not the dns.authorities field
+    # (authority resource records) is added to messages. The default is no.
+    include_authorities: yes
```

Comment:
I will change them to be true/false for consistency. Another project I work on uses yes/no for booleans so it happened out of habit.